### ISSUE TITLE
Files permissions issue on Windows 10 Fall Creators Update

### ISSUE THREAD

**Issue by jerome281** [npm/npm#19043]
Anyone having issues on Windows 10 Fall Creators Update? I'm having files permissions issues on the node_module folder, does n't seem to be able to write on that folder

npm ERR! Error: EPERM: operation not permitted
npm ERR!
npm ERR! Please try running this command again as root/Administrator.

#### I'm opening this issue because:

  - [x] npm is crashing.
  - [ ] npm is producing an incorrect install.
  - [ ] npm is doing something I don't understand.
  - [ ] Other (_see below for feature requests_):

### supporting information:

 - `npm -v` prints:  5.5.1
 - `node -v` prints: 9.0.0
 - `npm config get registry` prints:
 - Windows, OS X/macOS, or Linux?: Windows 10 Fall Creators Update
 - Network issues:
   - Geographic location where npm was run:
   - [ ] I use a proxy to connect to the npm registry.
   - [ ] I use a proxy to connect to the web.
   - [ ] I use a proxy when downloading Git repos.
   - [ ] I access the npm registry via a VPN
   - [ ] I don't use a proxy, but have limited or unreliable internet access.
 - Container:
   - [ ] I develop using Vagrant on Windows.
   - [ ] I develop using Vagrant on OS X or Linux.
   - [ ] I develop / deploy using Docker.
   - [ ] I deploy to a PaaS (Triton, Heroku).

**Comment by msroberts** [comment]
@jerome281 Yes, it looks like a lot of people are having this issue: #19042

I get in sporadically on Windows 7 and on every install on Windows 10 WSL.
